scripts/geoJSON.js

```diff
@@ -0,0 +1,49 @@
+var config = require('./config.js')
+var coords = require('./coords.js')
```

Comment:
Think about it like dependency injection. Rather than doing `require` in the same file it's used, do it in `index.js` and pass it into the functions that need it. That way each function only does one thing (unlike `renderMap()` which is also calling the function to create the geoJSON).